### ISSUE TITLE
Need a documentation to explain how to create Cloudflare credentials, just liked SST has for AWS

### ISSUE THREAD

**Issue by prabal-rje** [sst/ion#1220]
Here are some significant challenges when using Cloudflare with SST as a beginner:

- Cloudflare API tokens need to be created... but there's like a 100 permissions that may or may not be relevant...
- SST documentation (or the linked documentation from Cloudflare) doesn't explain the necessary ones required to get going with SST x Cloudflare integration.
- OTOH... AWS credential making process provided by SST (https://sst.dev/docs/iam-credentials#credentials) explains the EXACT steps necessary (including specific permissions) to create an AWS token.

Cheers,
PG

**Comment by jayair** [comment]
Yeah we are gonna take another pass at the Cloudflare setup soon.
